### ISSUE TITLE
Review overview/process

### ISSUE THREAD

**Issue by Dekor86** [MicrosoftDocs/microsoft-365-docs#9989]
The document I believe is outdated, refers to using XML files and deploying with Device Configuration profiles rather than the new Endpoint Security profiles. Also, multiple settings now available in settings catalog profile types vs existing "Templates" based Device Configuration profiles. Overview table contains links to other articles that are then not mentioned in the steps below, causing confusion on best practice for deploying Defender to macOS.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: cd66a255-36e5-29dc-7610-d50ef8bca3f3
* Version Independent ID: 17375473-bc07-f8e5-a85f-d732bbba3d71
* Content: [Intune-based deployment for Microsoft Defender for Endpoint on Mac](https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/mac-install-with-intune?view=o365-worldwide)
* Content Source: [microsoft-365/security/defender-endpoint/mac-install-with-intune.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/defender-endpoint/mac-install-with-intune.md)
* Service: **microsoft-365-security**
* Sub-service: **mde**
* GitHub Login: @Dansimp
* Microsoft Alias: **dansimp**

**Comment by yogkumgit** [comment]
@Dansimp Please help us in resolving this issue. Thanks

**Comment by emardee** [comment]
Agreed. I personally think this document is a little confusing in current form. The summary table and the more detailed steps below the table deviate in at least three ways that I spotted....

1.  The summary table includes the "Configure Microsoft AutoUpdate" which then isn't in the steps below the table. 
(Resolution would presumably be to add this to the steps below table too.)

2. The table also has the XML type of "Microsoft Defender for Endpoint Configuration Settings", but in the steps below, they talk about the Configuration Profiles instead (which I think is the correct method maybe?). 
(Resolution would presumably be to change Defender Configuration Settings line in the summary table to mention Configuration Profiles instead of XML method).

3. Finally, you have added the new Background Services step to the steps below the table, but this doesn't have a summary entry in the table. 
(Resolution would presumbaly be to add this as a line on the summary table too).

I am only just trying to decipher how this is all meant to work, so I am far from being an expert in this... so there is a reasonable chance my suggested resolutions above are not correct, but hopefully they at least help clarify what I believe needs looking at, even if the correct solution is actually different to my suggestion. Someone more knowledgable than me should be able to rectify it correctly.

Also, I'm glad to see that someone has fixed the wording a names for the Background Services section recently. Just before Christmas I was scratching my head, as that section had clearly been copy-pasted from the Notifications section, but hadn't been edited fully. (Maybe I just happened to be studying the page mid-edit, but glad to see that has been fixed already as that added to the confusion a few weeks ago!).

Hope this helps you to imprive the page.

**Comment by Baryczka** [comment]
Image link in step `Assign devices on the Assignment tab. Click Next.` is wrong
Current:
`source="images/mdatp-6-systemconfigurationprofiles-2.png"`
Expected:
`source="images/mdatp-6-systemconfigurationprofiles-3.png"`

Similar for the next step `Open Devices > Configuration profiles, you can see your created profile there.`
Current:
`source="images/mdatp-6-systemconfigurationprofiles-3.png"`
Expected:
`source="images/mdatp-6-systemconfigurationprofiles-4.png"`

**Comment by Baryczka** [comment]
**Full disk access** section might contain a wrong word as well. In the last sentence it should be:
"and downloaded fulldisk.mobileconfig as Configuration profile ~~name~~ file."

**Comment by officedocspr5** [comment]
To make it easier for you to submit feedback on articles on learn.microsoft.com, we're transitioning our feedback system from GitHub Issues to a new experience. 

As part of the transition, this GitHub Issue will be moved to a private repository. We're moving Issues to another repository so we can continue working on Issues that were open at the time of the transition. When this Issue is moved, you'll no longer be able to access it.

If you want to provide additional information before this Issue is moved, please update this Issue before December 15th, 2023.

With the new experience, you no longer need to sign in to GitHub to enter and submit your feedback. Instead, you can choose directly on each article's page whether the article was helpful. Then you can then choose one or more reasons for your feedback and optionally provide additional context before you select **Submit**.

Here's what the new experience looks like.

**Note**: The new experience is being rolled out across learn.microsoft.com in phases. If you don't see the new experience on an article, please check back later.

First, select whether the article was helpful:

![Image showing a dialog asking if the article was helpful with yes and no answers.](https://learn.microsoft.com/Office/media/internal/new-feedback-yes-no.jpg)

Then, choose at least one reason for your feedback and optionally provide additional details about your feedback:

| Article was helpful | Article was unhelpful |
|:--|:---|
|![Image showing a dialog asking how the article was helpful with several options.](https://learn.microsoft.com/Office/media/internal/new-feedback-helpful.jpg) | ![Image showing a dialog asking how the article wasn't helpful with several options.](https://learn.microsoft.com/Office/media/internal/new-feedback-unhelpful.jpg)|

Finally, select **Submit** and you're done!
